tempto-core/src/main/java/io/prestosql/tempto/AfterTestWithContext.java

```diff
@@ -12,7 +12,7 @@
  * limitations under the License.
  */
 
-package io.prestodb.tempto;
+package io.prestosql.tempto;
```

Comment:
I would squash that with the commit before (so it could have a chance to compile).

Reply:
This is intentionally separate so that moving files around is one commit and changing them is another. This helps Git track file renames.